values-changes.yaml

```diff
@@ -292,3 +292,7 @@ changes:
       - 'otomi.isHomeMonitored'
       - 'home'
       - 'dns.provider.civo'
+  - version: 27
```

Comment:
```suggestion
  - version: 29
```

Reply:
Note: run `npm run migrate-values` after this change to get fixtures updated.

Reply:
funny because `registry-controller` is not even supported because using `-` is not allowed ;-)